src/taipy/core/_core.py

```diff
@@ -96,6 +98,11 @@ def __manage_version(self, mode, _version_number, force, clean_entities):
 
             version_setter[mode](current_version_number, force)
 
+            if mode == "production":
+                collector = _MigrationConfigChecker(Config._applied_config, IssueCollector())._check()
```

Comment:
I would prefer to log the migrationConfig issues here. See remarks on [Config PR 56](https://github.com/Avaiga/taipy-config/pull/56). What do you think?